src/Stryker.Core/Stryker.Core/Mutators/InitializerMutator.cs

```diff
@@ -11,11 +11,12 @@ public class InitializerMutator : MutatorBase<InitializerExpressionSyntax>, IMut
 
         public override IEnumerable<Mutation> ApplyMutations(InitializerExpressionSyntax node)
         {
-            if (node.Parent is ArrayCreationExpressionSyntax || node.Parent is ImplicitArrayCreationExpressionSyntax || node.Parent is StackAllocArrayCreationExpressionSyntax)
+            if (node.Parent is ArrayCreationExpressionSyntax || node.Parent is ImplicitArrayCreationExpressionSyntax || node.Parent is StackAllocArrayCreationExpressionSyntax || node.Expressions.Count <= 0)
```

Comment:
Sorry, I'm late to the party (as usual), was just looking through the PRs now... Just to double-check, is this correct? Feels like >= should be here :) 

@rouke-broersma

Reply:
Should probably be `==`. The `>` case is handled in the ObjectCreationMutator.

@Liam-Rougoor typo?

Reply:
@psfinaki This is a check whether the node doesn't have any expressions. `==` would indeed have sufficed.

Reply:
Alright. I noticed it - I'll fix it :D